packages/web/src/app/Layout.tsx

```diff
@@ -1,15 +1,18 @@
-import { Outlet } from 'react-router-dom'
-import Header from './Header'
+import { Suspense } from 'react';
+import { Outlet } from 'react-router-dom';
+import Header from './Header';
 
 export function Layout() {
   return (
-    <div className="min-h-screen flex flex-col">
-      <Header />
-      <div className="flex-grow relative">
-        <Outlet />
+    <Suspense fallback="loading">
```

Comment:
This suspense will show when a component below needs to wait to render until something loads. At this point, that is only case when a locale file is loading.